apps/web/components/BackgroundImage/BackgroundImage.tsx

```diff
@@ -2,6 +2,7 @@ import React, { FC, useState, useEffect } from 'react'
 import cn from 'classnames'
 import { BoxProps, Box } from '@island.is/island-ui/core'
 import { theme, Colors } from '@island.is/island-ui/theme'
+import useIsMounted from '@island.is/web/hooks/useIsMounted'
```

Comment:
Would probably use [the hook](https://github.com/streamich/react-use/blob/master/docs/useMountedState.md) from `react-use` directly